cli/clibase/cmd.go

```diff
@@ -562,6 +562,16 @@ func Chain(ms ...MiddlewareFunc) MiddlewareFunc {
 	return chain(reversed...)
 }
 
+func UsageErrorPrefix(next HandlerFunc) HandlerFunc {
```

Comment:
I think this could be a bit more descriptive as `ShowUsageOnError` or something like that? LGTM otherwise.